chore: Updated readme to note carthage build setting (#3434)
aws-amplify/aws-sdk-ios: README.md

3. Then run the following command:
    
        $ carthage update --use-xcframeworks --no-use-binaries

> > As of Carthate 0.37.0, prebuilt binary framework is not supported with xcframework, as mentioned in the release changelog - https://github.com/Carthage/Carthage/releases/tag/0.37.0

Reviewer: ```suggestion
> As of Carthage 0.37.0, prebuilt binaries using XCFrameworks are not supported, as mentioned in the Carthage release notes - https://github.com/Carthage/Carthage/releases/tag/0.37.0
```
